lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -206,6 +209,12 @@ void SessionState::insert_far(SetGroupFAR* rule) {
   FarList_.push_back(*rule);
 }
 
+/* Remove all Pdr, FAR rules */ 
+void SessionState::remove_all_rules() {
```

Comment:
Do we not have to update the updatecriteria for this action?

Reply:
As we are doing .. in starting probably not required, later once we integrate with  policydb script then we might needed.

Reply:
Not in this case, as its called before we move to "created" state.  Not required, update creteria here.